programs/client/Client.cpp

```diff
@@ -1,31 +1,31 @@
+#include "Client.h"
```

Comment:
I guess you're using some plugin to sort includes, change format the code and all that after saving, but it's making the review much more difficult, apart of not being consistent with the rest of the code.

Is there any way to leave the code as it was and only apply formatting to your actual changes?

Reply:
I am applying clang-format according to ClickHouse's format, but I am not getting the supposed output.